corgi/core/files.py

```diff
@@ -65,21 +65,23 @@ class ProductManifestFile(ManifestFile):
     # We can subclass this to handle different Product subclasses with different properties
     # Or to handle different ways of generating manifest properties from Product properties
 
-    def render_content(self, cpe_mapping=True) -> str:
+    def render_content(self):
```

Comment:
Let's keep the return type hint:
```suggestion
    def render_content(self) -> str:
```

Reply:
ok